### PR TITLE
I was inspired this Saturday afternoon: Page H1 Titles and Heros now render in the header

### PR DESCRIPTION
**Pull request by cnrdgrgry**
so I'd been poking around the other teams sites and had inspiration from their code: as such page views no longer rely on pushed margins for the _title_ or _hero_ blurby bit

`useLocation() `and `case`  logic to determine the page and render the appropriate `H1` and `H3` was plugged into **SiteHeader.jsx** if you want to hunt for it

Then I had to go into all the page 'views' (i.e. home, about ...etc) and pull/comment out the associated .jsx and .css

Then readjusted all those negative margins to compensate.

Also improved display of popout burger-nav.

fingers in all the files.

NB @hedgehog125 @sfbennett  If you don't want to accept it without lots of changes , then we just don't need to bother, iIm not going to do any more this weekend , so if it's got lots of changes you want made, just reject it and we can leave the site as-is
**This is me playing, taken slow old me 5 hours! I think it's an improvement. approve or don't approve. I'm happy to just apply it to my fork after presentation otherwise.**

Screenshot from 2024-05-25 22-08-52
Screenshot 2024-05-25 at 22-09-43 PetSynth AI Assisted Companions
Screenshot 2024-05-25 at 22-09-25 PetSynth AI Assisted Companions
Screenshot from 2024-05-25 22-08-52

### CODE REVIEW

File: src/components/SiteHeader/SiteHeader.jsx [hunk 1]
```diff
@@ -1,13 +1,66 @@
 import React from "react";
+import { useLocation } from "react-router-dom";
 
 import SiteNav from "../SiteNav/SiteNav";
 import styles from "./SiteHeader.module.css";
 
 function SiteHeader() {
+	const location = useLocation();
+
+	console.log(location.pathname);
```
Comment (hedgehog125) [inline src/components/SiteHeader/SiteHeader.jsx:10]: Can I remove the console log though?

**Review by hedgehog125 — APPROVED** [review]
Yeah, that's looking a lot better. I think it might have already been fixed on desktop but it works much better on smaller phones like the iPhone SE now.

Before:
image

After:
image

Can I remove the console log though?

**Comment by cnrdgrgry** [comment]
Whoops did I leave that in? 

Go ahead if your in and poking around @hedgehog125  or a can in a bit.

**Review by sfbennett — APPROVED** [review]
👍

**Comment by hedgehog125** [comment]
Done

**Comment by cnrdgrgry** [comment]
Cool.
Thanks

Merging now, then I'll get into screenshots and doing the README.

**Pull request merged**